Dockerfile

```diff
@@ -13,7 +11,6 @@ RUN west update
 # West Zephyr export
 RUN west zephyr-export
```

Comment:
Oh, interesting. I combined these `RUN` lines but it did not make a difference in the image size; apparently it isn't doing too much in the way of transient files? I also don't know enough about `west` to say which of those commands are likely to bust the layer cache, so I'm inclined to keep them individually cache-able.

Reply:
Ha, my very next run demonstrated this in action:
```
 => CACHED [4/7] RUN west init -l config
 => [5/7] RUN west update
```
The rule you linked to makes the tradeoff clear and I'm ok optimizing for speed in this case.
> Putting easy cacheable and non-cacheable commands in the same layer (by using one RUN command) might have negative performance issues.

Reply:
O sorry, it shouldn't do anything for the size/speed, should just effect the layers that Docker generates. I was thinking it may be a case similar to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#apt-get (with subcommands like init and update).
Can avoid issues where earlier stages succeed & cached, resulting in later stages to have a(n) inconsistent/un-reproducible/non-updating state. I.e. they're actually coupled stages and should be apart of the same `RUN`